DB/Pets/Dragonflight.lua

```diff
@@ -119,6 +119,48 @@ local dragonflightPets = {
 		chance = 50,
 		coords = { { m = CONSTANTS.UIMAPIDS.OHN_AHRAN_PLAINS, x = 55.2, y = 76.7, L["Force of the Springs"] } },
 	},
+	["Scruffles"] = {
+		cat = CONSTANTS.ITEM_CATEGORIES.DRAGONFLIGHT,
+		type = CONSTANTS.ITEM_TYPES.PET,
+		method = CONSTANTS.DETECTION_METHODS.NPC,
+		name = L["Scruffles"],
+		npcs = { 200584 },
+		itemId = 193364,
+		spellId = 375047,
+		chance = 25,
+		creatureId = 189115,
+		groupSize = 5,
+		equalOdds = true,
+		coords = { { m = CONSTANTS.UIMAPIDS.THE_FORBIDDEN_REACH, x = 58.6, y = 48.8, n = L["Vraken the Hunter"] } },
+	},
+	["Wakyn"] = {
+		cat = CONSTANTS.ITEM_CATEGORIES.DRAGONFLIGHT,
+		type = CONSTANTS.ITEM_TYPES.PET,
+		method = CONSTANTS.DETECTION_METHODS.NPC,
+		name = L["Wakyn"],
+		npcs = { 200960 },
+		itemId = 191930,
+		spellId = 371594,
+		chance = 20,
+		creatureId = 188821,
+		groupSize = 5,
+		equalOdds = true,
+		coords = { { m = CONSTANTS.UIMAPIDS.THE_FORBIDDEN_REACH, x = 45.9, y = 79.7, n = L["Warden Entrix"] } },
```

Comment:
Also missing in L?

Reply:
i believe they should have been added in with a different mount?

Reply:
#539 these dont need to be added in a 2nd time right?

Reply:
Oh, right. I didn't realize they were the same rares - never mind then!